app/assets/stylesheets/govuk_publishing_components/components/_breadcrumbs.scss

```diff
@@ -20,3 +20,27 @@
 .gem-c-breadcrumbs--inverse .govuk-breadcrumbs__list-item:before {
   border-color: govuk-colour("white");
 }
+
+@include govuk-media-query($until: tablet) {
+  .govuk-breadcrumbs__list {
+    display: flex;
+  }
+
+  .govuk-breadcrumbs__list-item {
+    display: none;
+  }
+
+  .gem-c-breadcrumbs__list-item--first-last {
+    display: block;
+  }
+
+  .govuk-breadcrumbs__list-item:before {
+    top: 5px;
+    bottom: auto;
+  }
+
+  .govuk-breadcrumbs__list,
+  .govuk-breadcrumbs__link {
```

Comment:
I wonder if we should consider increasing touch area sizes upstream in GOV.UK Frontend?

Reply:
Might need some more testing around this. Completely unscientifically I've just tried the existing breadcrumb on my phone and had no trouble with it, but that's just me.